### DISCUSSION TITLE
Uncaught TypeError: window.Alpine.cloneNode is not a function

### DISCUSSION THREAD

**Discussion by cloudwales** [livewire/livewire#6238] [1. Help] [answered]
Hi. I was working on an app where everything was working, but all of a sudden I get this error in the console:

'Uncaught TypeError: window.Alpine.cloneNode is not a function'

I went back a few commits in github incase it was something I done, but I get the same issue. 

Any ideas.

**Comment by cloudwales** [comment]
So i found the error. Not sure why this was the case, but nothing I had added.

In app.js

From this:
```
import './bootstrap';
window.Alpine = Alpine
Alpine.start()
```

to this:
```
import './bootstrap';
// window.Alpine = Alpine
// Alpine.start()
```

And it all worked as it should.

**Reply by StringKe** [reply]
```ts
import './bootstrap';
import Alpine from 'alpinejs';

import Typewriter from '@marcreichel/alpine-typewriter';

window.document.addEventListener('livewire:init', () => {
    console.log('livewire:init');
    window.Alpine.plugin(Typewriter);
});

declare global {
    interface Window {
        Alpine: typeof Alpine;
    }
}
```

try this, Alpinejs initialized by livewire

**Reply by joshhanley** [reply, marked as answer]
@cloudwales and @StringKe please read the Livewire V3 upgrade guide regarding Alpine https://livewire.laravel.com/docs/upgrading#including-via-js-bundle

<img width="759" alt="image" src="https://github.com/livewire/livewire/assets/882837/567e2581-10da-4924-9f80-1b941e43bb4b">

**Reply by StringKe** [reply]
@joshhanley Do I still need `@livewireScripts` in the blade to use this approach?

**Reply by joshhanley** [reply]
@StringKe no, you need to use `@livewireScriptConfig`. Please read the upgrade guide, all the details should be there 🙂

<img width="771" alt="image" src="https://github.com/livewire/livewire/assets/882837/534e6017-c9b1-4e64-8ec6-7ca74f01e996">

**Reply by StringKe** [reply]
Okay, I've tested it.

It's already working properly, thank you.

**Reply by joshhanley** [reply]
@StringKe no worries! 🙂

**Comment by princevora** [comment]
You can just use the followings in a same file

@livewireScripts 
 @livewireScriptConfig
